dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jaxwsClient-2.2.feature

```diff
@@ -8,7 +8,7 @@ IBM-Process-Types: client
  com.ibm.websphere.appserver.containerServices-1.0, \
  com.ibm.websphere.appserver.classloading-1.0, \
  com.ibm.websphere.appserver.javax.mail-1.5; ibm.tolerates:="1.6", \
- com.ibm.websphere.appserver.jaxb-2.2
+ com.ibm.websphere.appserver.jaxb-2.2; ibm.tolerates:=2.3
```

Comment:
I think the `ibm.tolerates:=2.3` is fine for now - while we're developing JAXB/JAX-WS 2.3, but once the development of JAX-WS 2.3 gets a little further along, we should remove the tolerates so that JAX-WS 2.2 is linked with JAXB 2.2, and JAX-WS 2.3 is linked with JAXB 2.3.  Other (non-JAX-WS) features may still need to tolerate both.